config/config.go

```diff
@@ -0,0 +1,56 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/USACE/filestore"
+	_ "github.com/jackc/pgx/stdlib"
```

Comment:
You can remove this import, you aren't connecting to postgres here. You should also be able to remove any postgres libraries from the go.mod.

Reply:
Great, catch! Done.